src/modules/site-v2/templates/_scripts/utils.js

```diff
@@ -108,6 +105,23 @@ function create_node(html) {
 }
 
 
+function save_svg(selector, filename=null) {
```

Comment:
consider setting the default value for filename as a default argument, eg:
`save_svg(selector, filename = 'download.svg') {`